web/front/js-json-extra/translation.json

```
@@ -173,7 +173,35 @@
         "Three-Way operator" : false
     },
 
-    
+    "OCaml" : {
+        "Variable declaration" : "let s = ref \"a variable\"",
+        "Constant declaration" : "let s = \"a constant\"",
+        "Function declaration" : "let a_function argument = argument",
+        "Lambda declaration" : "a_lambda = fun argument -> argument",
+        "While loop" : "while b do\n\t...\ndone",
+        "For loop" : "for i = 0 to n - 1 do\n\t...\ndone",
+        "Foreach loop" : "for elem in a_list do\n\t...\ndone",
+        "Do .. while loop" : false,
+        "Foreach index and element loop" : false,
+        "Infinite loop" : "while true do\n\t...\ndone",
+        "if ... else if ... else" : "if b1 then\n\t...\nelse if b2 then\n\t...\nelse\n\t...",
+        "Ternary operator" : "if b then _true else _false",
+        "Try ... Catch" : "try \n\t...\ncatch e\n\t...\nend",
+        "Switch" : "match x with | y -> ... | _ -> ...",
+        "Type of the %" : "Truncated",
+        "Top level constructor" : false,
+        "Get a line of STDIN" : "read_line()",
+        "Get all STDIN" : false,
+        "Get a char STDIN" : "input_char stdin",
+        "Print without a new line at the end" : "print_string s",
+        "Print with a new line at the end" : "print_endline s",
+        "Implicit conversion of bool to int" : false,
+        "Implicit conversion of char to int" : false,
+        "Format a string" : "Printf.printf \"%d\" n",
```

```suggestion
        "Format a string" : "Printf.sprintf \"%d\" n",
```
To keep it consistent with the rest.